Source/WTF/wtf/text/StringHash.h

```diff
@@ -237,6 +237,12 @@ namespace WTF {
         {
             return a == b;
         }
+
+        static void translate(String& location, StringView view, unsigned hash)
+        {
+            location = view.toString();
```

Comment:
A little sad that when the `StringView` covers an entire `String`, we will construct a new `String` and can’t use the `String` it was made from. We could fix this by introducing a type that contains both a `StringView` and a reference to an underlying `String`, but we need not do that in this patch. It might be tricky to design this so it’s not cumbersome or risky to use.